internal/network/command_dns_record.go

```diff
@@ -37,16 +38,7 @@ func (c *command) newDnsRecordCommand() *cobra.Command {
 }
 
 func (c *command) addAccessPointFlag(cmd *cobra.Command) {
-	cmd.Flags().String("access-point", "", "PrivateLink access point.")
-}
-
-func (c *command) getDnsRecords() ([]networkingaccesspointv1.NetworkingV1DnsRecord, error) {
-	environmentId, err := c.Context.EnvironmentId()
-	if err != nil {
-		return nil, err
-	}
-
-	return c.V2Client.ListDnsRecords(environmentId)
+	cmd.Flags().String("private-link-access-point", "", "PrivateLink access point.")
```

Comment:
Why is it "PrivateLink" in this description, but "Private Link" in the table output?

Reply:
Oversight.